docs/messages_guide/src/asciidoc/_chapters/ddl_msgs.adoc

```diff
@@ -254,6 +254,24 @@ local super ID can perform a replication operation for a catalog and for
 multiple schemas in a single catalog. Additionally, the schema owner can
 perform a replication operation for a single schema.
 
+[[SQL-1018]]
+== SQL 1018
+
+```
+Grant of role or privilege <privilege> from <user-or-role-1> to <user-or-role-2> not found, revoke request ignored.
+```
+
+Where <privilege> is a {project-name} privilege and <user-or-role-1> and <user-or-role-2> are user IDs or roles.
+
```

Comment:
I didn't see PUBLIC mentioned anywhere in the MESSAGES guide. But I've tried some wordsmithing which you'll see in the next draft. Let me know (then) if you think it is appropriate.